tardis/montecarlo/tests/test_rpacket.c

```diff
@@ -0,0 +1,16 @@
+#include "../src/rpacket.h"
+#include <assert.h>
+#include <stdlib.h>
+
+void test_rpacket_get_nu(){
+	rpacket_t * rp = (rpacket_t *) malloc(sizeof(rpacket_t));
+	double value = 10.2;
+	rpacket_set_nu(rp, value);
+	assert(value == rpacket_get_nu(rp));
```

Comment:
you need to write a code that raises a python exception when this test fails:

```python
if value != rpacket_get_nu(rp){ python assert}
``` - I don't know how maybe @juliantaylor knows.